packages/components/src/Toast/Toast.test.tsx

```diff
@@ -36,49 +35,70 @@ const infoMessage = "Bland Toast";
 const errorMessage = "Errorful should last inbetween min-max";
 
 it("creates the placeholder div on showToast call", () => {
-  const { getByText } = render(<MockToast />);
-  expect(document.querySelector(`#atlantis-toast-element`)).not.toBeInstanceOf(
-    HTMLDivElement,
-  );
+  const { getByText, queryByText, getByLabelText } = render(<MockToast />);
 
   fireEvent.click(getByText("No Variation"));
 
-  expect(document.querySelector(`#atlantis-toast-element`)).toBeInstanceOf(
-    HTMLDivElement,
-  );
+  expect(getByText(infoMessage)).toBeInstanceOf(HTMLSpanElement);
```

Comment:
Atlantis have `@testing-library/jest-dom` installed so you can just check if it's in the dom
```suggestion
  expect(getByText(infoMessage)).toBeInTheDocument()
```